Resources/skeleton/crud/views/show.html.twig.twig

```diff
@@ -13,10 +13,18 @@
             <tr>
                 <th>{{ field|capitalize }}</th>
 
-            {%- if metadata.type in ['date', 'datetime'] %}
+            {%- if metadata.type in ['datetime'] %}
 
                 <td>{{ '{% if ' ~ entity_singularized ~ '.' ~ field|replace({'_': ''}) ~ ' %}{{ ' ~ entity_singularized ~ '.' ~ field|replace({'_': ''}) ~ '|date(\'Y-m-d H:i:s\') }}{% endif %}' }}</td>
 
+            {%- elseif metadata.type in ['date'] %}
+
+                <td>{{ '{% if ' ~ entity_singularized ~ '.' ~ field|replace({'_': ''}) ~ ' %}{{ ' ~ entity_singularized ~ '.' ~ field|replace({'_': ''}) ~ '|date(\'Y-m-d\') }}{% endif %}' }}</td>
+            
+            {%- elseif metadata.type in ['date'] %}
```

Comment:
you are right. Fixed in https://github.com/sensiolabs/SensioGeneratorBundle/commit/342dea51f4c40e90ce45a227da02b90f7e0aeba3  Thanks.